### PR TITLE
Include applicant ID in program CSV export

### PR DESCRIPTION
**Pull request by bion**
- [x] include applicant ID as a default column in program CSV export
- [x] mount sbt cache directories in sbt docker container for faster builds
- [x] rename a few enums for clarity

column type enum renaming:

`APPLICANT` -> `APPLICANT_ANSWER`
`ID` -> `APPLICATION_ID`

and added a new column type of `APPLICANT_ID`

fixes https://github.com/seattle-uat/civiform/issues/1735

### CODE REVIEW

File: bin/sbt [hunk 1]
```diff
@@ -10,5 +10,8 @@ bin/pull-image
 docker run -it --rm \
   -v /var/run/docker.sock:/var/run/docker.sock \
   -v $(pwd)/universal-application-tool-0.0.1:/usr/src/universal-application-tool-0.0.1 \
-  civiform \
+  -v ~/.coursier:/root/.coursier \
```
Comment (shanemc-goog) [inline bin/sbt:13]: Looks unrelated.

**Review by shanemc-goog — APPROVED** [review]

**Comment by elisekalstad** [comment]
Oh my goodness, this is amazing! Thank you @shanemc-goog  and @bion !
